### PR TITLE
Add bash commands syntax highlighting

### PR DESCRIPTION
**Pull request by gustavoteixeirah**
## Risks and Area of Effect
- [ ] Is this a breaking change?

## Testing
- [ ] Unit test
- [ ] Deployed to dev env
- [ ] Other (explain) 

## Reverting
- [ ] Contains Migration - _Do Not Revert_

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,19 +4,19 @@ The quickstart example is a document summarization bot that can take a file, URL
...
 * Setup and activate python virtual environment
-   - brew install pyenv-virtualenv 
-   - python3 -m venv yourownenv
-   - source yourownenv/bin/activate
+   - `brew install pyenv-virtualenv`
+   - `python3 -m venv yourownenv`
+   - `source yourownenv/bin/activate`
 * Install streamlit
-   - Pip install streamlit
+   - `pip install streamlit`
```
Comment (jkwatson-verta) [inline README.md:11]: doesn't this come along when you install the requirements.txt?

File: README.md [hunk 2]
```diff
@@ -4,19 +4,19 @@ The quickstart example is a document summarization bot that can take a file, URL
...
 * Download or git-clone the quickstart project
-* Go to main folder of the project and run pip install -r requirements.txt
-* Go to src folder and run streamlit run LLM_Builder.py
-* The  quickstart app will open in localhost
+* Go to main folder of the project and run `pip install -r requirements.txt`
+* Go to src folder and run `streamlit run LLM_Builder.py`
```
Comment (jkwatson-verta) [inline README.md:14]: it would be better to get the env vars set up before this, I think.

File: README.md (hunk 1 above)
Comment (gustavoteixeirah) [reply README.md:11]: Yes it does... I wasn't sure if there's a reason for preinstalling it or not, that's why I didn't removed  it

File: README.md (hunk 2 above)
Comment (gustavoteixeirah) [reply README.md:14]: Agreed

**Review by jkwatson-verta — APPROVED** [review]
Thanks!

**Pull request merged**